oecophylla/cli/launch.py

```diff
@@ -49,18 +49,54 @@ def _create_dir(_path):
         os.makedirs(_path)
 
 
+def _oeco_dir():
+    d = "%s/../../" % os.path.abspath(os.path.dirname(__file__))
+
+    if not os.path.isdir(d):
+        raise OSError('Cannot find Oecophylla install directory'
+                      '(tried %s)' % d)
+    else:
+        oeco_dir = os.path.abspath(d)
+        return oeco_dir
+
+def _setup_test():
+    d = _oeco_dir()
+
+    # set inputs
+    input_dir = os.path.join(d, 'test_data/test_reads')
+    sample_sheet = os.path.join(d,
+                        'test_data/test_config/example_sample_sheet.txt')
+    # set params
+    params = os.path.join(d,
```

Comment:
if `os.path.join` looks bit to long, we can do `from os.path import join` then simply do `join`. This won't conflict with Python's native string `join`.